config/pipeline.yaml

```diff
@@ -130,10 +130,10 @@ storage:
     port: 8022
     base_url: http://172.17.0.1:8002/
 
-  early-access-azure: &azure-files
+  production-azure: &azure-files
     storage_type: azure
     base_url: https://kciapistagingstorage1.file.core.windows.net/
-    share: early-access
+    share: production
```

Comment:
@nuclearcat Could you please confirm if updating `share` value won't break storage?